Mage.Sets/src/mage/cards/m/MaskwoodNexus.java

```diff
@@ -128,6 +119,16 @@ public boolean apply(Game game, Ability source) {
                 game.getState().getCreateMageObjectAttribute(card, game).getSubtype().setIsAllCreatureTypes(true);
             }
         }
+
+        // Apply to all effected cards
+        affectedCards.stream().map(card -> game.getObject(card.getId())).filter(Objects::nonNull).forEach(mageObject -> {
+                    game.getState().getCreateMageObjectAttribute(mageObject, game).getSubtype().setIsAllCreatureTypes(true);
+                    CardUtil.getObjectParts(mageObject).stream().filter(Objects::nonNull).forEach(objectId ->{
```

Comment:
That’s wrong. Must filter each part for creature. There are possible lands, instances and other parts in mdf.

Reply:
Moved filters